Makefile

```diff
@@ -39,6 +39,7 @@ image-hmi-client: clean-hmi-client yarn-install
 	yarn workspace graph-scaffolder tsc --build
 	yarn workspace hmi-client build
 	mv $(PROJECT_DIR)/packages/client/hmi-client/dist $(PROJECT_DIR)/packages/client/hmi-client/docker/dist
+	cp ${PROJECT_DIR}/packages/client/hmi-client/silent-sso.html $(PROJECT_DIR)/packages/client/hmi-client/docker/dist/silent-sso.html
```

Comment:
this should be part of the vite config

`build: {
		target: 'esnext',
		rollupOptions: {
			input: {
				main: path.resolve(__dirname, 'index.html'),
				sso: path.resolve(__dirname, 'silent-sso.html')
			}
		}
	},`